personal/Adam/Reports/zajecia2.Rmd

```diff
@@ -0,0 +1,59 @@
+---
+title: "Zajecia2"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Zadanie 1
+
+Wyświetlcie mapę krajów Europejskich  wg liczby ludności
+
+
+
+```{r cars}

```

Comment:
ten blok nie powinien się nazywać cars ;) . Rozumiem, że to pozostałość z automatycznego szablonu, ale mimo to należałoby to zmienić.